### PR TITLE
Flavour Fuse_BMM

### PR DESCRIPTION
**Pull request by Maria-Manuela**
## Netlify link
https://flavourfus.netlify.app/

## Collaborators
Solo Project

### CODE REVIEW

File: index.html
Comment (Antontall) [inline index.html:1]: Looks nice overall, to the point. The indentation looks off though.

File: script.js
Comment (Antontall) [inline script.js:1]: Nice, nothing here looks redundant as far as I can tell. The comment ´//function to handle the click event´ could be clearer and more descriptive, like ´call random recipe function´ or something. Again I think the indentation looks off.

**Comment by Antontall** [comment]
The page has a nice layout and all the functions works but I noticed the stir fry recipe has null as cooking time. Well done! :)

File: index.html [hunk 1]
```diff
@@ -3,11 +3,63 @@
...
+    </select>
+
+    <p>Pick for me!</p>
+    <div>
+      <button id="random">Random</button>
+    </div>
+  </div>
+ 
+ 
+
```
Comment (NellaMori) [inline index.html:47]: Please format your code to have the right spaces and indentation 👍

File: script.js
Comment (NellaMori) [inline script.js:1]: Please format your code to have the right spaces and indentation 👍

**Review by NellaMori — APPROVED** [review]
Hi Maria-Manuela, well done finishing your Library app 📚 You followed the instructions and the app fulfils the requirements of filtering and responsiveness.
I left some comments in the code to improve the code structure by formatting, overall you did a good job 🚀

**Pull request closed without merging**